implementation/discovery/etcd/src/main/java/org/inaetics/dronessimulator/discovery/etcd/EtcdChangeHandler.java

```diff
@@ -54,6 +65,14 @@ public void run() {
         Logger.getLogger(EtcdChangeHandler.class).info("Stopped EtcdChangeHandler!");
     }
 
+    /**
+     * Adds handlers for node events.
```

Comment:
They are added to the root of the tree so they will receive all respective events.